ArduCopter/Parameters.cpp

```diff
@@ -53,7 +53,7 @@ const AP_Param::Info Copter::var_info[] = {
     // @Param: SYSID_MYGCS
     // @DisplayName: My ground station number
     // @Description: Allows restricting radio overrides to only come from my ground station
-    // @Values: 255:Mission Planner and DroidPlanner, 252: AP Planner 2
+    // @Values: 252: AP Planner 2, 255:Mission Planner and DroidPlanner
```

Comment:
I wonder if we shouldn't just remove these entirely.

Reply:
I agree since the rest of the vehicles don't have it either. Instead, they have a @range field in them.  Do you think I should replace it with a range field to make it uniform?

Reply:
The ID 252 of APM PLANNER2 is written in the code.
The FC can identify the GCS.
If you set it to RANGE, it would be nice to have a description of the setting value in the description of the item.
I have an automatic configuration parameter reference generation.

Reply:
@muramura I have removed it, for now, just to make it the same as the other vehicles. Let's raise another PR and have the description changed as you want for all the SYSID_MYGCS in all the vehicles. Thanks :)